README.md

```diff
@@ -28,6 +28,23 @@ let package = Package(
 )
 ```
 
+Or, if you are using Swift 5.2:
+
+```swift
+// swift-tools-version:5.2
+import PackageDescription
+
+let package = Package(
+  name: "MyTool",
+  dependencies: [
+    .package(name: "SwiftSyntax", url: "https://github.com/apple/swift-syntax.git", .exact("<#Specify Release tag#>")),
```

Comment:
Is this syntax not working for SPM versions before 5.2? If it does work, why not have this be the 'canonical' version, instead of showing 2 different versions?

Reply:
Good point. I somehow assumed it wouldn’t work before Swift 5.2 but it does. Changed the main template instead.